Documentation/cmdref/cilium-agent.md

```diff
@@ -300,6 +300,8 @@ cilium-agent [flags]
       --monitor-queue-size int                                    Size of the event queue when reading monitor events
       --mtu int                                                   Overwrite auto-detected MTU of underlying network
       --multicast-enabled                                         Enables multicast in Cilium
+      --nat-map-stats-entries int                                 Number k top stats entries to store locally in statedb (default 32)
+      --nat-map-stats-interval duration                           Interval upon which nat maps are iterated for stats (default 30s)
```

Comment:
Should we also say that `duration = 0` disables the stats collection? Or is that obvious?